actions/s3_pull/action.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Pull PyPi Wheel"
+description: "Pull PyPi wheel"
+
+inputs:
+  filename:
+    description: 'File to pull from s3 bucket'
+    required: true
+  dst:
+    description: 'Folder to pull down wheel file'
+    required: false
+    default: true 
+runs:
+  using: "composite"
+  steps:
+    - name: Push to s3
+      shell: bash
+      run:  |
+            aws s3 cp ${{ inputs.filename }} ${{ inputs.dst }}
```

Comment:
i've shied away from using s3 in runners for public repos, but we'll just need to be careful about permissions and such.